### DISCUSSION TITLE
Why is the configuration not automatically moved when upgrading to 1.2.x?

### DISCUSSION THREAD

**Discussion by alessio-b2c2** [python-poetry/poetry#6823] [General]
Hi, I just upgraded to poetry 1.2.2 from the previous minor version.

I now see this warning:
```
❯ poetry --version
Configuration file exists at /Users/alessio/Library/Application Support/pypoetry, reusing this directory.

Consider moving configuration to /Users/alessio/Library/Preferences/pypoetry, as support for the legacy directory will be removed in an upcoming release.
Poetry (version 1.2.2)
```

My question is, why doesn't poetry automatically do this for me? I expect any other application of my system to deal with this automatically, not bother me with moving the configuration files around.

**Comment by alessio-b2c2** [comment]
Also, if I do as requested, then `poetry --version` doesn't work anymore:

```
❯ mv "/Users/alessio/Library/Application Support/pypoetry" /Users/alessio/Library/Preferences/pypoetry

❯ poetry --version
zsh: command not found: poetry
```

**Comment by neersighted** [comment]
There was strong opposition to an automatic migration at the time, so a warning was added. The intention is to introduce an automatic migration in a future version, as I recall. The reason Poetry broke is you cannot move the folder wholesale (the messaging could definitely be better there). In that folder you will find `venv`, `config.toml`, and `auth.toml`. Only the `*.toml` files need to move -- the venv needs to stay there as the install layout has not changed, only the config layout.
